src/card.css

```
@@ -0,0 +1,77 @@
+.card {
+  padding: 16px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  padding: 16px 3px 40px;
+}
+
+.card__description {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.reviews {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-bottom: 16px;
+}
+
+.price-text {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  color: #616070;
```

don't forget to create variables for all colors
```suggestion
  color: $c-color;
```